apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -146,3 +146,8 @@ div.pl-order-blocks-header {
 .pl-order-blocks-pairing-indicator li:last-child {
   margin-bottom: 0px !important;
 }
+
+.selected:focus {
+  outline: 3px solid #ff6200;
+  outline-offset: -3px;
+}
```

Comment:
Our convention is to prefix all element CSS with the name of the element, so this should be something like `pl-order-block-selected`